### DISCUSSION TITLE
BT-809 in SDK 1.12.0 inconsistent xpaths in guidance and sdk files

### DISCUSSION THREAD

**Discussion by odscjen** [OP-TED/eForms-SDK#990] [Q&A] [answered]
In the latest SDK [fields.json](https://github.com/OP-TED/eForms-SDK/blob/develop/fields/fields.json) BT-809 has
> xpathAbsolute: /*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:SelectionCriteria/cbc:TendererRequirementTypeCode

But in the[ SDK docs table 6](https://docs.ted.europa.eu/eforms/latest/schema/procedure-lot-part-information.html) its XSD element is
> ext:UBLExtensions​/ext:UBLExtension​/ext:ExtensionContent​/efext:EformsExtension​/efac:SelectionCriteria​/cbc:CriterionTypeCode

From the rest of the guidance and the mention of this field in the [SDK 1.12.0](https://github.com/OP-TED/eForms-SDK/releases/tag/1.12.0) release notes I'm assuming the fields.json is incorrect and the docs are correct but could you please confirm?

cc @jpmckinney

**Comment by YvesJo** [comment]
Thanks for reporting that concern.
I have to investigate and will let you know more.

**Reply by YvesJo** [reply]
The major shift introduced by the new approach on Selection Criteria, with a univocal reference to a selection criterion (BT-809) instead of a selection criteria category (BT-747), required the introduction of a new element.
Unfortunately the documentation does not reflect that change and this will get fixed.

**Reply by bertrand-lorentz** [reply, marked as answer]
The online documentation has now been corrected, and the corrected version is available on the linked page:
https://docs.ted.europa.eu/eforms/latest/schema/procedure-lot-part-information.html#tenderingTermsRelatedInformationTable
